### PR TITLE
Support Browser API event mozbrowserconnected

### PR DESCRIPTION
**Pull request by paulrouget**
Fixes https://github.com/servo/servo/issues/9382

This new event is not yet documented. If this lands, I will add documentation to MDN.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9740)

### CODE REVIEW

**Comment by paulrouget** [comment]
> This pull request adds a file without the .ini file extension to tests/wpt/mozilla/meta. Please consider removing it!

I don't understand.

**Comment by KiChjang** [comment]
False positive. Currently highfive doesn't know that MANIFEST.json can be ignored. There's an ongoing PR that fixes this in servo/highfive#62.

File: components/script/dom/document.rs [hunk 1]
```diff
@@ -2589,9 +2597,6 @@ impl DocumentProgressHandler {
 
         document.notify_constellation_load();
 
-        // https://developer.mozilla.org/en-US/docs/Web/Events/mozbrowserloadend
-        document.trigger_mozbrowser_event(MozBrowserEvent::LoadEnd);
```
Comment (jdm) [inline components/script/dom/document.rs]: Why are we getting rid of this, and how does the test still pass?

**Comment by jdm** [comment]
```
176.40s$ bash etc/ci/manifest_changed.sh

diff --git a/tests/wpt/mozilla/meta/MANIFEST.json b/tests/wpt/mozilla/meta/MANIFEST.json

index bf051fe..b12f912 100644

--- a/tests/wpt/mozilla/meta/MANIFEST.json

+++ b/tests/wpt/mozilla/meta/MANIFEST.json

@@ -5928,18 +5928,18 @@

             "url": "/_mozilla/mozilla/mozbrowser/iframe_goback.html"

           }

         ],

-        "mozilla/mozbrowser/mozbrowser_loadevents.html": [

-          {

-            "path": "mozilla/mozbrowser/mozbrowser_loadevents.html",

-            "url": "/_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html"

-          }

-        ],

         "mozilla/mozbrowser/iframe_reload_twice.html": [

           {

             "path": "mozilla/mozbrowser/iframe_reload_twice.html",

             "url": "/_mozilla/mozilla/mozbrowser/iframe_reload_twice.html"

           }

         ],

+        "mozilla/mozbrowser/mozbrowser_loadevents.html": [

+          {

+            "path": "mozilla/mozbrowser/mozbrowser_loadevents.html",

+            "url": "/_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html"

+          }

+        ],

         "mozilla/mozbrowser/mozbrowsericonchange_event.html": [

           {

             "path": "mozilla/mozbrowser/mozbrowsericonchange_event.html",
```

**Comment by KiChjang** [comment]
@bors-servo retry
* #9488

**Pull request merged**